superset/databases/schemas.py

```diff
@@ -307,6 +307,12 @@ class DatabaseValidateParametersSchema(Schema):
         allow_none=True,
         validate=server_cert_validator,
     )
+    configuration_method = EnumField(
```

Comment:
I think @AAfghahi is going to add in a Meta class with `unknown = EXCLUDE` in this validation schema class. In that case, we can remove this.